.github/workflows/essence-feature-stats.yml

```diff
@@ -12,9 +12,13 @@ on:
 env:
   ESSENCE_DIR: "./EssenceCatalog"
```

Comment:
does this still need to be in the ENV? did you do multiple repos yet in this PR?

Reply:
Yes! Repos are specified in another variable (ESSENCE_FILE_REPOS I think?)
This is just the root directory for essence files, all repos will be cloned to its sub directories. I think it doesn’t matter for GitHub actions but this variable is still expected to be set

Reply:
is it not a bit confusing to have it set to EssenceCatalog, which is one of the repo names?

Reply:
true, changed that